### DISCUSSION TITLE
Trouble with index variables

### DISCUSSION THREAD

**Discussion by Nicoolai** [nguyenquyhy/Flight-Tracker-StreamDeck#341] [Q&A] [answered]
Hola all,
I'm trying to use variables with indexes, for Feedback values.
Example is `ENG COMBUSTION:1==1` to see if my engine is on.
Every variable I use, with index, seems to not work for me though.
Am I doing something wrong in my formatting?

**Comment by nguyenquyhy** [comment, marked as answer]
Try `GENERAL ENG COMBUSTION:index`. Right now the variable has to be in https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/FlightStreamDeck.Core/TOGGLE_VALUE.cs to be supported.

**Reply by Nicoolai** [reply]
Worked like a charm!
Thank you. I will reference that file in the future for sure.
